lmdeploy/pytorch/supported_models.py

```diff
@@ -45,11 +45,15 @@
     # cogvlm-chat
     CogVLMForCausalLM=True,
     # llava
-    LlavaLlamaForCausalLM=False,
+    LlavaLlamaForCausalLM=True,
```

Comment:
Should we add `LlavaMistralForCausalLM` as well?

Reply:
yes